src/js/apps/patients/patient/flow/flow_app.js

```
@@ -58,7 +58,7 @@
     }
     
     /* eslint-disable no-console */
-    console.error(arguments);
+    console.error(response, responseData = {}, ...arguments);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)